pkg/cache/sql/store/store.go

```diff
@@ -203,7 +216,7 @@ func (s *Store) List() []any {
 func (s *Store) ListKeys() []string {
 	rows, err := s.QueryForRows(context.TODO(), s.listKeysStmt)
 	if err != nil {
-		fmt.Printf("Unexpected error in store.ListKeys: %v\n", err)
+		fmt.Printf("Unexpected error in store.ListKeys: %v\n", fmt.Errorf("while executing query: %s got error: %w", s.listKeysQuery, err))
```

Comment:
Why not compose this as a single string?

It's now using `%v` which will trigger `.Error()` on the `err` so it could just be a single `fmt.Printf(...)`?

Reply:
Right, thanks for the observation. Unfortunately commits in this PR were merged together as part of #84, so I am starting [a separate PR](https://github.com/rancher/lasso/pull/90) to address this feedback